src/main/java/org/codewars/utils/App.java

```diff
@@ -0,0 +1,8 @@
+package org.codewars.utils;
+
+public class App {
+
+    public static void main(String[] args) {
+        
+    }
```

Comment:
mb just use org/codewars/App.java ?

Reply:
on this path, the App file already exists. I can specify a different name